periflow/cli/checkpoint.py

```diff
@@ -634,6 +615,64 @@ def restore(
     typer.secho(f"Checkpoint({checkpoint_id}) is successfully restored.")
 
 
+@app.command("import")
+@check_api
+def import_from_catalog(
+    catalog_name: str = typer.Argument(
+        ...,
+        help="The name of public checkpoint to try out.",
+    ),
+    name: str = typer.Option(
+        None,
+        "--name",
+        "-n",
+        help="The name of the checkpoint that will be created in the project.",
+    ),
+    method: CatalogImportMethod = typer.Option(
+        CatalogImportMethod.COPY.value,
+        "--method",
+        "-m",
+        help=(
+            "The method to import the public checkpoint. When the method is 'copy', "
+            "the file objects will be copied to a separate storage so that the "
+            "checkpoint remains available even if the source public checkpoint is "
+            "removed from the catalog. Conversely, when the 'ref' method is selected, "
+            "the file objects are referenced from the storage of the source public "
+            "checkpoint. In this case, the imported checkpoint becomes unavailable if "
+            "the source in the catalog is deleted."
+        ),
```

Comment:
Since REF has a side effect when the public checkpoint in the catalog is deleted.
For the same reason, in the web application, the default (and also recommended) method is COPY too.